src/index.html

```diff
@@ -1,13 +1,367 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>B&O</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.png"
+    >
+    <link href="https://cdn.jsdelivr.net/npm/reset-css@5.0.1/reset.min.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__content">
+    <header class="header page__section">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo">
+          <img src="./images/logo.png" alt="Bang&Olufsen logo">
+        </a>
+
+        <div class="top-bar__icons">
+          <a href="tel:+1-234-5555-55-55" class="icon icon--phone icon--with-text">
```

Comment:
You need to fix format number phone only in hrer everywhere
```suggestion
          <a href="tel:+123455555555" class="icon icon--phone icon--with-text">
```